### DISCUSSION TITLE
Network Overview/Diagram not loading

### DISCUSSION THREAD

**Discussion by tir599** [netalertx/NetAlertX#909] [Q&A]
Earlier in my deployment this used to load but now just spins, how can I troubleshoot it? It makes the lowers sections of this page unusable as well.
![image](https://github.com/user-attachments/assets/61c57775-5ded-449d-9c7b-cdc55c0ea8a6)

**Comment by jokob-sk** [comment]
Hi,

Did you by chance create a recursion? E.g.  assigning a node to itself? (e.g. similar issue: https://github.com/jokob-sk/NetAlertX/issues/905)

If that's the case, can you remember what you did so I can prevent this from happening? 

If above doesn't help, can you check also the browser console debug log?

Thanks,
j

**Reply by tir599** [reply]
I don't see anything recursive and that thread didn't help. I do not see any errors or warnings in logs.

My best guess is that I may have renamed and added MAC/IP to the default 'Internet' node since it showed offline and my pfSense is on an isolated vLAN. I was able to setup SNMP on it later.

**Reply by tir599** [reply]
Is there still a way to create a dummy device like in the docs? I do not have that option.

If not, how do I blow everything away so that it recreates that device from scratch?

**Reply by tir599** [reply]
NM, I was able to get it through .csv import and map is working again.
Is the icmp_scan plugin gone? I'd love to be able scan/get online devices on my other routable subnets that are not wifi (Unifi plugin).

**Reply by jokob-sk** [reply]
Hey, the ICMP plugin is available in the netalertx-dev container and will be available in the next stable release.

Glad that the issue is resolved.
J

Same for adding dummy devices. You can find upcoming changes in this issue https://github.com/jokob-sk/NetAlertX/issues/138
